test/models/cmr_test.rb

```diff
@@ -79,13 +79,27 @@ class CmrTest < ActiveSupport::TestCase
+      stub_request(:get, "https://cmr.earthdata.nasa.gov/search/collections.echo10?page_num=1&page_size=10&provider=NSIDCV0&provider=ORNL_DAAC&provider=LARC_ASDC&provider=LARC&provider=LAADS&provider=GES_DISC&provider=GHRC&provider=SEDAC&provider=ASF&provider=LPDAAC_ECS&provider=LANCEMODIS&provider=NSIDC_ECS&provider=OB_DAAC&provider=CDDIS&provider=LANCEAMSR2&provider=PODAAC&keyword=*doris*").with(:headers => {'Accept' => '*/*', 'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent' => 'Ruby'}).to_return(:status => 200, :body => "HTTP/1.1 200 OK
+Access-Control-Allow-Origin: *
```

Comment:
I think this was just supposed to be the response body, but you pasted the response status and headers in here too...